modules/google-analytics/README.md

```diff
@@ -0,0 +1,113 @@
+# WP Google Analytics
```

Comment:
This file probably won't be needed in the plugin. The contents would make a good support doc though. :)